tests/serverpod_test_server/test_e2e/signout/auth_signout_test.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/test_key_manager.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
+  group('Given an authenticated user', () {
+    late Client client;
+    late Client secondClient;
+
+    setUpAll(() async {
```

Comment:
Question: We are not validating any status endpoints in this file. Do we really need to to the method validation through the client? Couldn't we just do this as integration tests on the server?

Reply:
Noticed now that `auth_status_signout_test.dart` is basically the same file. I think that one has more value than this file if there is no difference between them.

Reply:
Removed this.